src/Sylius/Bundle/CoreBundle/Workflow/Listener/OrderShipping/AfterOrderShippedListener.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Workflow\Listener\OrderShipping;
+
+use Sylius\Bundle\CoreBundle\Workflow\Callback\OrderShipping\OrderShippedCallbackInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Symfony\Component\Workflow\Event\Event;
+use Webmozart\Assert\Assert;
+
+final class AfterOrderShippedListener
+{
+    /** @param OrderShippedCallbackInterface[] $callbacks */
+    public function __construct(private iterable $callbacks)
+    {
+        Assert::allIsInstanceOf($callbacks, OrderShippedCallbackInterface::class);
+    }
+
+    public function call(Event $event): void
```

Comment:
```suggestion
    public function call(CompletedEvent $event): void
```
https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Workflow/Event/CompletedEvent.php